CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ## StreamChat
+### ✅ Added
+- Logout from `ChatClient` [#2241](https://github.com/GetStream/stream-chat-swift/pull/2241)
```

Comment:
It's better to provide more information here, something like:
"From now on, if you want to log out the user from the app, especially when switching users, you should call the client.logout() instead of client.disconnect(). Read more [here](link)"